app.rb

```diff
@@ -0,0 +1,119 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  attr_accessor :people, :rentals, :books
+
+  def initialize
+    @people = []
+    @rentals = []
+    @books = []
+  end
+
+  def book_list
+    if @books.empty?
+      puts 'There is no book in the list'
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index} - Title: #{book.title.capitalize}, Author: #{book.author.capitalize}"
+      end
+    end
+  end
+
+  def people_list
+    if people.empty?
+      puts 'There is no person in the list'
+    else
+      @people.each_with_index do |person, index|
+        if person.instance_of?(Teacher)
+          puts "#{index} [Teacher] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+        elsif person.instance_of?(Student)
+          puts "#{index} [Student] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+        end
+      end
+    end
+  end
+
+  def person_create
+    puts 'Do you want to create a student (1) or a teacher (2)? [input the number]:'
+    type = gets.chomp
+    case type
+    when '1'
+      student_create
+    when '2'
+      teacher_create
+    else
+      puts 'Invalid selection'
+    end
+  end
+
+  def teacher_create
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Specialization:'
+    specialization = gets.chomp
+    parent_permission = true
+    @people.push Teacher.new(age: age, name: name, specialization: specialization, parent_permission: parent_permission)
+    puts 'Person created successfully'
+  end
+
+  def student_create
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Has parent permission? [y/n]:'
+    permission = gets.chomp.downcase
+    parent_permission = permission == 'y'
+    puts 'Classroom:'
+    classroom = gets.chomp
```

Comment:
- [ ] @kimitaWanjohi I can see you added an option for a student to enter their class, however, it's not part of the requirements in the console app design shown in the sneak peek, I would highly recommend sticking to the design given which is part of the project requirement.
![image](https://github.com/kimitaWanjohi/school_lib/assets/61408860/ad2fd37e-2d7c-482e-a597-6fb14bb02f47)